lib/ii.c

```diff
@@ -12962,14 +13066,19 @@ grn_ii_builder_register_chunks(grn_ctx *ctx, grn_ii_builder *builder)
 
   a = array_get(ctx, builder->ii, builder->chunk.tid);
   if (!a) {
+    grn_obj token;
+    grn_ii_get_token_from_token_id(ctx, builder->ii, builder->chunk.tid, &token);
     DEFINE_NAME(builder->ii);
     MERR("[ii][builder][chunk][register] "
          "failed to allocate an array in segment: "
          "<%.*s>: "
-         "tid=<%u>: max_n_segments=<%u>",
+         "tid=<%u>: max_n_segments=<%u>: "
+         "token:<%.*s>",
          name_size, name,
          builder->chunk.tid,
-         builder->ii->seg->header->max_segment);
+         builder->ii->seg->header->max_segment,
+         GRN_TEXT_LEN(&token), GRN_TEXT_VALUE(&token));
```

Comment:
We should cast `GRN_TEXT_LEN(&token)` to `int` to suppress a warning on some environment.

Reply:
https://travis-ci.org/groonga/groonga/jobs/369929306#L1549

> ```text
> ii.c:2912:7: warning: field precision specifier ‘.*’ expects argument of type ‘int’, but argument 7 has type ‘long int’ [-Wformat=]
>       MERGE_BC(bid.rid <= rid || cid.rid);
```

Did you see this warning on your environment?
       ^